cypress/support/cypress-support/src/main.js

```diff
@@ -253,14 +254,36 @@ export default function (module) {
   Cypress.Commands.add('sendMessagetoPlatforms', (requestMap) => {
     cy.wrap(requestMap, { timeout: CONSTANTS.SEVEN_SECONDS_TIMEOUT }).then(async (requestMap) => {
       return new Promise(async (resolve) => {
-        const message = await config.getRequestOverride(requestMap);
-        // perform MTC call/FB call only if the message is not null
-        if (message != null) {
-          const response = await transport.sendMessage(message);
-          const result = config.getResponseOverride(response);
-          resolve(result);
+        const [moduleName, methodName] = requestMap.method.split('.');
+        Cypress.env(CONSTANTS.REQUEST_OVERRIDE_METHOD, methodName);
+        // Check if request is for fcs setters
+        if (moduleName === 'fcsSetter') {
+          const method = fcsSetter[methodName];
+          if (method && typeof method === 'function') {
+            try {
+              const { attribute, value } = requestMap.params;
+              const response = await method(attribute, value);
+              resolve(response);
+            } catch (error) {
+              resolve(
+                setterFailure(`Error while invoking ${requestMap.method} due to ${error.message}`)
+              );
+            }
+          } else {
+            resolve(setterNotImplemented(`Method ${methodName} not implemented`));
```

Comment:
We don't want to resolve here either. We can reject if we need to do something but we shouldn't be resolving

Reply:
Addressed